erpnext/public/js/order_for.js

```diff
@@ -0,0 +1,8 @@
+frappe.ready(function () {
+
+    $('.order-for-contact-logout').click(function() {
+        frappe.call("erpnext.utilities.order_for.reset_website_customer").then(r => {
+          window.location.href = r.message;
+        });
+      });
```

Comment:
- Add this is cart.js

Reply:
Neil had decided to put this in a separate file, I'll bring it up with him when we work on the the edits